### DISCUSSION TITLE
Conditional if author sign in or sign out

### DISCUSSION THREAD

**Discussion by swanteadikrisna** [BoidCMS/BoidCMS#37] [General]
Hello, i'm interested in this cms. Can i request conditional if author sign in or sign out ?

**Comment by sysadev** [comment]
Hi, welcome to the community 😊

And yes, BoidCMS offers conditional logic based on user login status through the `App::logged_in` property. See API Reference: https://boidcms.github.io/#/developer/reference

**Comment by DenizToprakYildiz** [comment]
eyw my buddy
